sale_cutoff_time_delivery/models/sale_order.py

```diff
@@ -58,6 +58,9 @@ class SaleOrderLine(models.Model):
     def _prepare_procurement_values(self, group_id=False):
         """Postpone delivery according to cutoff time"""
         res = super()._prepare_procurement_values(group_id=group_id)
+        return self._sale_cutoff_time_delivery_prepare_procurement_values(res)
```

Comment:
Looking at the function below nothing will be returned from `_prepare_procurement_values` if there is a  `date_planned` in res.
This could lead to issues, maybe ?

Reply:
There's actually a trick here, you can click on this button, which will unfold the code that hadn't changed :smile: 
![image](https://user-images.githubusercontent.com/13200247/133738897-881aebfe-07b8-42df-b775-46aced788e42.png)

Reply:
:rofl: I need to find my :eyeglasses: